Submitted By: Joe Ciccone <jciccone@gmail.com>
Date: 2009-01-04
Initial Package Version: 2.6.28
Origin: http://www.kernel.org/pub/linux/kernel/v2.6/snapshots/patch-2.6.28-git5.log
Upstream Status: From Upstream
Description: Adds a missing include to the sanatized serial.h header.

diff -Naur linux-2.6.28.orig/include/linux/serial.h linux-2.6.28/include/linux/serial.h
--- linux-2.6.28.orig/include/linux/serial.h	2008-12-24 18:26:37.000000000 -0500
+++ linux-2.6.28/include/linux/serial.h	2009-01-04 22:09:52.136007007 -0500
@@ -10,8 +10,8 @@
 #ifndef _LINUX_SERIAL_H
 #define _LINUX_SERIAL_H
 
-#ifdef __KERNEL__
 #include <linux/types.h>
+#ifdef __KERNEL__
 #include <asm/page.h>
 
 /*
